docs/what-is-ROCmValidationSuite.rst

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Should this file be removed - don't see it in the landing page.

Reply:
Done.